core/pom.xml

```diff
@@ -188,5 +188,6 @@ Bundle-DocURL:
             <groupId>uk.org.lidalia</groupId>
             <artifactId>slf4j-test</artifactId>
         </dependency>
```

Comment:
@nitheshkumar-hcl  Why we need this dependency?

Reply:
It's already existing in development branch,didn't added it as part of this PR.

Reply:
@nitheshkumar-hcl  can you remove this file from PR.?